pages/user/project.js

```diff
@@ -0,0 +1,163 @@
+import UserHeader from '../components/user_header';
+
+export default function Signup() {
+  return (
+    <div>
+      <UserHeader />
+      <section className="mt-3">
+        <div className="container">
+          <div className="row u-section">
+            <div className="column column-25 f">
+              <div className="current m project">
+                <div className="row r">
+                  <img src="/assets/images/avatar-1.png" alt="project-logo" />
+                </div>
+                <div className="row r">
+                  <h5>Project Name</h5>
+                </div>
+                <hr className="hr" />
+                <div className="row">
+                  <div className="column">
+                    <h6>Organization Name</h6>
+                    <p>Catergory: Educationi</p>
+                    <p>Cost: 5 Million FCFA</p>
+                    <p>Date Started: 12th Jan 2020</p>
+                    <p>Duration: 6Months</p>
+                    <p>
+                      Status: <span className="in-progress">In Progress</span>
+                    </p>
+                    <p>
+                      Website: <a href="projectwebsite.com">www.projectwebsite.com</a>
+                    </p>
+                    <p>2,345 Followers</p>
+                  </div>
+                </div>
+              </div>
+              <div className="complete m">
+                <div className="row pr project">
+                  <div className="row">
+                    <input type="checkbox" id="follow-project" />
+                    <p htmlFor="follow-project">Follow Project</p>
+                  </div>
+                  <div className="row">
+                    <input type="checkbox" id="upvote" />
+                    <p htmlFor="upvote">Upvote Project</p>
+                  </div>
+                  <div className="row">
+                    <input type="checkbox" id="downvote" />
+                    <p htmlFor="downvote">Downvote Project</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="column column-40 m">
+              <div className="row r">
+                <h5>About Organization</h5>
+              </div>
+              <hr className="hr" />
+              <div className="row d">
+                <p>
```

Comment:
## Nitpick
Can we abstract the dumb text into a file so that the code can be cleaner & readable?